channels/offline-channel-js/README.md

```diff
@@ -0,0 +1,29 @@
+# Microsoft Application Insights JavaScript SDK - Offline Channel
+
+## Description
+
+The Offline Channel supports the saving of events when your application is offline and resending those events when the application is online.
+
+### Note
+
+- A post or sender channel is required for processing online events.
+- Request header details will be stored in local/session storage or IndexedDB based on your configuration.
+- If you are using the default Breeze endpoint `https://dc.services.visualstudio.com` for Application Insights, partial success is currently considered as success and events not sent in partial success will be dropped.
```

Comment:
Breeze is the internal name this should be either Applicatin Insights or Azure Monitor